CustomBlockPlugin.inc.php

```diff
@@ -115,21 +115,5 @@ function getContents($templateMgr, $request = null) {
 		return parent::getContents($templateMgr, $request);
 
 	}
-
-	/**
-	 * @copydoc BlockPlugin::getBlockContext()
-	 */
-	function getBlockContext() {
-		if (!Config::getVar('general', 'installed')) return BLOCK_CONTEXT_SIDEBAR;
-		return parent::getBlockContext();
-	}
-
-	/**
-	 * @copydoc BlockPlugin::getSeq()
-	 */
-	function getSeq($contextId = null) {
```

Comment:
Are you sure this is no longer used? I haven't dug into the main PR yet, but until now, this has been the mechanism that orders block plugins on the sidebar.

Reply:
Yeah, we now use a `sidebar` row in `journal_settings` to indicate which blocks are active and in what order they should appear.